client/App.jsx

```diff
@@ -9,16 +9,21 @@ React.Component {
         super(props);
         this.state = {
             selectedItem: 1,
-            numOfImgs: 6,
+            numOfImgs: 2,
             itemName: "",
             numOfVids: 0,
             imageNumber: 1
         }
-        this.get.bind(this);
-        this.onClick.bind(this);
+        this.getImageData.bind(this);
+        this.onClickSide.bind(this);
+        this.onClickMain.bind(this);
     }
 
-    get() {
+    componentDidMount() {
+        this.getImageData();
+    }
+
+    getImageData() {
```

Comment:
Actually a really good idea to have this funtion separate from componentDidMount so that it can be reused, I'm doing the same with one of mine.